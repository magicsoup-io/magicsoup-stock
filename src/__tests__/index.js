import React, { Fragment } from 'react'
import renderer from 'react-test-renderer'
import {
  Box,
  Button,
  Card,
  Flex,
  Heading,
  Image,
  Link,
  Text,
} from '../index'

it('imports magicsoup.io without errors', () => {
  const tree = renderer.create(
    <Fragment>
      <Box> I am a Box</Box>
      <Button> I am a Button</Button>
      <Card> I am a Card</Card>
      <Flex> I am a Flex</Flex>
      <Heading> I am a Heading</Heading>
      <Image src="https://source.unsplash.com/random/1280x720" />
      <Link href="http://www.example.com"> I am a Link</Link>
      <Text> I am a Text</Text>
    </Fragment>,
  ).toJSON()
  expect(tree).toMatchSnapshot()
})
