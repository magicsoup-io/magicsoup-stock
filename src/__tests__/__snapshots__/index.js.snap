// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`imports magicsoup.io without errors 1`] = `
Array [
  <div
    className="sc-bdVaJa eAkHCv"
  >
     I am a Box
  </div>,
  <button
    className="sc-bdVaJa sc-bwzfXH hLpsMy"
    color="white"
    fontSize="inherit"
    fontWeight="bold"
  >
     I am a Button
  </button>,
  <div
    className="sc-bdVaJa sc-htpNat jocheC"
  >
     I am a Card
  </div>,
  <div
    className="sc-bdVaJa sc-ifAKCX cGFghZ"
  >
     I am a Flex
  </div>,
  <h2
    className="sc-bdVaJa sc-EHOje sc-bZQynM kKcxZB"
  >
     I am a Heading
  </h2>,
  <img
    className="sc-bdVaJa sc-gzVnrw gWAdUb"
    src="https://source.unsplash.com/random/1280x720"
  />,
  <a
    className="sc-bdVaJa sc-htoDjs ckKdfW"
    color="blue"
    href="http://www.example.com"
  >
     I am a Link
  </a>,
  <div
    className="sc-bdVaJa sc-EHOje ilTtvD"
  >
     I am a Text
  </div>,
]
`;
